Add Community Events page + update Marketing ownership (#11543)
PostHog/posthog.com: contents/handbook/words-and-pictures/events.md

---
title: Events
sidebar: Handbook
showTitle: true
---

> > If you’re planning an event and want support or promotion from the PostHog team, go to [Community Events](https://posthog.com/handbook/words-and-pictures/events#community-events)

Reviewer: ```suggestion
> If you’re planning an event and want support or promotion from the PostHog team, go to [Community Events](/handbook/words-and-pictures/events#community-events)
```

Won't correct them all, but tiny FYI that internal links like this are best as relative paths

> We’re [100% remote](https://posthog.com/handbook/company/culture) and set up the company to work asynchronously–but there's a real benefit in getting together in real life (and our best ideas come from [working together](https://posthog.com/handbook/company/offsites)).

Reviewer: ```suggestion
We’re [100% remote](https://posthog.com/handbook/company/culture) and set up to work asynchronously–but there's a real benefit in getting together in real life (and our best ideas come from [working together](https://posthog.com/handbook/company/offsites)).
```

Reviewer 2: ```suggestion
We’re [100% remote](https://posthog.com/handbook/company/culture) and set up to work asynchronously, but there's a real benefit in getting together in real life (and our best ideas come from [working together](https://posthog.com/handbook/company/offsites)).
```

> Still, we’re skeptical about how much value events deliver. We prefer to be a small fish in a big pond, so we pass on conferences. We [prefer pull](https://posthog.com/handbook/growth/marketing#2-pull-dont-push) over push, so we avoid formats like webinars, dinners, and lunch & learns.

Reviewer: ```suggestion
Still, we’re skeptical about how much value events deliver. We prefer to be a small fish in a big pond, so we pass on big conferences. We [prefer pull](https://posthog.com/handbook/growth/marketing#2-pull-dont-push) over push, so we avoid formats like webinars, dinners, and lunch & learns.
```

> Still, we’re skeptical about how much value events deliver. We prefer to be a small fish in a big pond, so we pass on conferences. We [prefer pull](https://posthog.com/handbook/growth/marketing#2-pull-dont-push) over push, so we avoid formats like webinars, dinners, and lunch & learns.

Reviewer 2: The first sentence of this is a bit of a negative frame to this whole doc, even though it might be true. Maybe this instead?
```suggestion
Still, we’re figuring out how to do events "the PostHog way." We prefer to be a small fish in a big pond, so we pass on big conferences. We [prefer pull](https://posthog.com/handbook/growth/marketing#2-pull-dont-push) over push, so we avoid formats like webinars, dinners, and lunch & learns.
```

Author: Yes! Like this much better. Thank you!

  - When [product engineers](https://posthog.com/blog/what-is-a-product-engineer) get together to identify problems and build solutions for users

## Community Events

People often underestimate the power and potential within their teams and ecosystems. Community events are in real life (IRL) manifestations of [our mission](https://posthog.com/handbook/why-does-posthog-exist#our-mission) organized by enthusiastic customers. They take place when someone has identified a gap or opportunity where an event helps people move [faster](https://posthog.com/newsletter/this-is-why-youre-not-shipping), smarter, and more together.

We avoid limiting event potential, so we don’t have a template to follow. With that said, some considerations:

  - **Structure:** participatory > nonparticipatory, bottoms-up > top down. unconference > conference.
  - **Curation:** Get the right mix of people to the event. Prioritise this alongside the content. This extends to speakers, co-hosts, and attendees.
  - **Atmosphere** foster a [welcoming environment](https://posthog.com/handbook/company/grown-ups#things-we-do-to-create-a-welcoming-environment) where everyone can succeed.

> In our high trust environment we expect organizers who initiate events to fully own them.

Reviewer 2: ```suggestion
In our high trust environment, we expect organizers who initiate events to fully own them.
```

> What Community Events are **not** for:

Reviewer 2: ```suggestion
What community events are **not** for:
```

  - Forcing PostHog or any other product into conversations with people
  - Watching or planning things rather than doing them (Workshops > Fireside Chats)
  - Networking

### Formulating a purpose and structure

> An impactful event follows the principles of user-driven development which stems from user problems or requests. Who is the user for your Community Event? The user can be your actual customers, fellow founders, local engineers or any other collection(s) of people. Talk to them first to validate if the event is worth your time.

Reviewer 2: ```suggestion
An impactful event follows the principles of user-driven development which stems from user problems or requests. Who is the user for your community event? The user can be your actual customers, fellow founders, local engineers or any other collection(s) of people. Talk to them first to validate if the event is worth your time.
```

> We use GitHub for everything at PostHog. When you’re ready to organize an event, create a GitHub issue [using this template](https://github.com/PostHog/meta/issues/new?template=event-plan.md) and assign it to Daniel Zaltsman. Prioritize progress (on which you can build) over perfection.

Put real effort into this first step. Defining the "what, why and how" of an event beforehand will pay off on event day. Let our [shared values](https://posthog.com/handbook/values) guide you. Don’t submit until your answer to “Would I attend this?” is a clear “yes.”

For an event to meet the high bar of quality for our work, we’ve provided considerations for ways to make it achieve its purpose within the GitHub issue template.

### Getting support

> We support Community Events led by proactive organizers with a bias for action.

Reviewer 2: ```suggestion
We support community events led by proactive organizers with a bias for action.
```

**Co-promotion:** Most of the time the help requested is in the form of promotion. As a general rule, we don't promote events we aren't supporting or co-hosting ourselves. We decide when to repost Community Events on our social media channels on a case by case basis. To boost your chances of co-promotion:

  - Words are spelled (correctly) using American English and you’ve tagged PostHog
  - It includes a link to the event page, which includes an agenda and relevant information
  - There is some form of multimedia: meta image, image, gif, or video

**Venue and catering:** Identify the vendors and costs and include them in the GitHub issue. If the event will not be possible without monetary support, make that clear. We may support the cost of venue, food, or beverages but require the paper napkin math.

**Feedback:** You’ll learn more by doing than planning so don’t worry about having every detail complete before submitting for feedback from our team.

### Branding it

Our brand is a reflection of us and [how we’re experienced by others](https://posthog.com/blog/brand), including events.

> **Words:** For our customers to have a clear sense of our involvement in Community Events, avoid including PostHog in the title. The same goes for referring to it as a “PostHog Event/Meetup/Gathering” with or without the word "Official." Accepted alternatives are "PostHog-supported," “with support from the PostHog team,” or “with PostHog’s support.”

Reviewer 2: ```suggestion
**Words:** For our customers to have a clear sense of our involvement in community events, avoid including PostHog in the title. The same goes for referring to it as a “PostHog Event/Meetup/Gathering” with or without the word "Official." Accepted alternatives are "PostHog-supported," “with support from the PostHog team,” or “with PostHog’s support.”
```

**Pictures:** Every event is improved with a flyer or poster that showcases the essence of the experience. We keep a comprehensive list of brand assets and guidelines for their use on the dedicated [brand assets page](https://posthog.com/handbook/company/brand-assets). Share your assets and we’ll give feedback. Depending on the scale and timing of the event, our team may be able to help with branding as well.

### Follow-ups

> Community Events are better when organizers share what happened and any follow-up actions.

Reviewer 2: ```suggestion
Community events are better when organizers share what happened and any follow-up actions.
```
